.env.test

```diff
@@ -22,4 +22,7 @@ TRELLO_BOARD_ID=TEST_TRELLO_BOARD_ID
 
 PRIMARY_CONNECTION_STRING_DB=mongodb://fakemongo.com
 
-DB_NAME=chaostodoSaved
+chaostodoSaved=chaostodoSaved
```

Comment:
I don't see any evidence that you actually use this environment variable - does that sound right?

Bonus - it would be good to add the keys for `GITHUB_AZ_CLIENT_ID` and `GITHUB_AZ_CLIENT_SECRET` to your `.env.template` file so that users can easily see they need to set those

Reply:
Another bonus - I'd naturally guess the `AZ` stands for Azure, but our OAuth has no direct knowledge about Azure, so is that a misnomer?